### PR TITLE
Clarify correct way to find kafkfa ID in the Prometheus prerequisites

### PR DESCRIPTION
**Pull request by fmcdonal**

### CODE REVIEW

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 1]
```diff
@@ -201,9 +201,11 @@ As an alternative to viewing metrics for a Kafka instance in the {product-long-k
...
-* You have the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots), and click *Connection*.
-* You have the generated credentials for your service account that has access to the Kafka instance. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
+* You've access to a running Kafka instance that contains topics in {product-kafka}. For more information about access management in {product-kafka}, see {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
+* You've the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots) and click:
```
Comment (grdryn) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:205]:
```suggestion
* You've the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To locate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots) and click: 
```

**Review by grdryn — APPROVED** [review]
Looks good to me, thanks! I've left one minor suggestion inline. Also, I don't have the expertise to know which is preferred between "You have" and "You've". I probably would have defaulted to "You have" and I probably would have been wrong :)

File: docs/kafka/metrics-monitoring-kafka/README.adoc (hunk 1 above)
Comment (AedinC) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:205]:
```suggestion
* You have the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots) and click: 
```

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 2]
```diff
@@ -201,9 +201,11 @@ As an alternative to viewing metrics for a Kafka instance in the {product-long-k
...
-* You have the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots), and click *Connection*.
-* You have the generated credentials for your service account that has access to the Kafka instance. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
+* You've access to a running Kafka instance that contains topics in {product-kafka}. For more information about access management in {product-kafka}, see {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
+* You've the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots) and click: 
+** *Details* to locate the ID which is the ID of the Kafka instance.
+** *Connection* tab to locate the SASL/OAUTHBEARER Token endpoint URL.
```
Comment (AedinC) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:207]:
```suggestion
** *Connection* tab to locate the SASL/OAUTHBEARER token endpoint URL.
```

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 3]
```diff
@@ -201,9 +201,11 @@ As an alternative to viewing metrics for a Kafka instance in the {product-long-k
...
+* You've the generated credentials for your service account that has access to the Kafka instance. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
```
Comment (AedinC) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:208]:
```suggestion
* You have the generated credentials for your service account that has access to the Kafka instance. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
```

**Review by AedinC — COMMENTED** [review]
Thanks Frances-just a few minor comments.

File: docs/kafka/metrics-monitoring-kafka/README.adoc (hunk 2 above)
Comment (fmcdonal) [reply docs/kafka/metrics-monitoring-kafka/README.adoc:207]: @AedinC I used upper case "T" for token because that is the way it appears in the UI

**Review by AedinC — APPROVED** [review]
LGTM

**Pull request merged**